OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -140,9 +147,11 @@ public void Tick(Squad owner)
 				}
 			}
 
+			// Switch target during fight
```

Comment:
While I disabled individual units from scanning the closest target, the group can still switch target